### DISCUSSION TITLE
[QUESTION]

### DISCUSSION THREAD

**Discussion by dylan-kwon** [patrykandpatrick/vico#315] [Questions]
When there is less data, the line is aligned in the center. 
Is there a way to align it similar to the SpaceBetween alignment of Row?

### Current
<img width="428" alt="스크린샷 2023-07-20 오전 9 36 57" src="https://github.com/patrykandpatrick/vico/assets/21299684/3c84794d-fb92-45a2-a621-3cd6cdec72b2">

### I want
<img width="420" alt="스크린샷 2023-07-20 오전 9 35 50" src="https://github.com/patrykandpatrick/vico/assets/21299684/fd2c3232-b66c-4891-8f3c-e50204092187">

**Module**
- [x] `compose`
- [ ] `views`
- [ ] `compose-m2`
- [x] `compose-m3`
- [ ] `core` (select if none of the above apply)

**Comment by patrickmichalik** [comment]
Hello! Switching from `HorizontalLayout.Segmented` to `HorizontalLayout.FullWidth` will produce this result. All top-level elements take a `HorizontalLayout` instance.
